gambling.js

```diff
@@ -17,22 +18,56 @@ exports.exportLedger = function() {
 }
 
 /**
+ * returns an 's' iff count > 1.
+ * 
+ * @param {number} count 
+ */
+function maybeGetPlural(count) {
+    if (count > 1)
+        return 's';
+    return '';
+}
+
+/**
+ * Adds the given number of Scrubbing Bubbles to the provided user's army.
+ * 
+ * @param {String} userID - id of the user to add to
+ * @param {Number} amount - amount to add
+ * @param {String} addTake - Whether to Add or Take from Army
+ */
+function addToArmy(userID, amount, addTake) {
```

Comment:
First of all why is a function called addToArmy able to take from an army? second of all why are you passing a string in, when you don't actually need to use that string in the function. you might as well be using a boolean. this is known as using flags and generally considered very bad design. conditional complexity could probably be improved.